portfolio/src/main/webapp/script.js

```diff
@@ -12,16 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
```

Comment:
I think Toya deleted this in a previous PR - just be careful when merging. Normally I'd ask to rebase or merge with main and update the PR but in the interests of time, this is okay.